rdt/transformers/numerical.py

```diff
@@ -90,6 +107,12 @@ def reverse_transform(self, data):
         if self.nan is not None:
             data = self.null_transformer.reverse_transform(data)
 
+        if self.max_value is not None:
```

Comment:
I think using `clip` here would make it simpler and faster.

```
data = data.clip(self._min_value, self._max_value)
```

Also, this should probably need to be done before the `null_transformer.reverse_transform`, otherwise null values will make it crash

Reply:
@csala After running the timer tests for this way of clipping vs `clip` I found that the time returned for `clip` on an array of 1000 values was `0.06289540000034322` and the time returned for this custom clipping was `0.061169600000084756`. This shows the custom clipping is slightly faster, but I don't know if it's fast enough to justify the extra complication

Reply:
Yes, being such a small difference I would go for the simpler `clip`